src/smbclient/shutil.py

```diff
@@ -295,7 +293,10 @@ def copytree(
     :param kwargs: Common arguments used to build the SMB Session for any UNC paths.
     :return: The dst path.
     """
-    dir_entries = list(scandir(src, **kwargs))
+    if is_remote_path(src):
+        dir_entries = list(scandir(src, **kwargs))
+    else:
+        dir_entries = [LocalDirEntry(os.path.join(src, result)) for result in os.listdir(src)]
```

Comment:
using `os.scandir` is most likely going to be more efficient and should theoretically remove the need for `LocalDirEntry`.